dist/main.js

```diff
@@ -15948,6 +15879,13 @@ angular.module('znk.infra-web-app.socialSharing').run(['$templateCache', functio
                     return defer.promise;
                 };
 
+                // Load stripe module <script src="https://checkout.stripe.com/checkout.js"></script>
+                function loadStripeScript() {
+                    const scriptElm = $window.document.createElement('script');
+                    scriptElm.src = STRIPE_SCRIPT_URL;
+                    $window.document.body.appendChild(scriptElm);
```

Comment:
try what I did in wis2biz should be better performance rather then append child

Reply:
fixed